docs/kratos/bring-your-own-ui/01_overview.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-id: custom-ui-overview
+id: index
```

Comment:
Unfortunately, this doesn't work (the file needs to be named index too). At the moment, this page is unavailable:

https://www.ory.sh/docs/kratos/bring-your-own-ui/index
https://www.ory.sh/docs/kratos/bring-your-own-ui

Please revert this change!